Project-3/README.md

```diff
@@ -11,67 +11,238 @@ For any question involving the use of Protege, please be sure to import:
+  (2) states: “it is declared, i.e. something can only be a characteristic of one thing.” What one might infer from this declaration is that this was done for           pragmatic purposes in order to elicit preferred inference relations without having to deal with the burden of unwanted inferences to irrelevant elements. So for       example, by declaring a functional constraint on “characteristic of” the ontologist could make pragmatic use of a delimited ontology that always garners a one to one   relation, rather than a one to many, and so on. As a result, if the reasoner is run the functional constraint set will restrict the domain to one to one functional     relations only. Thereby granting the conditions for a more specific, restricted ontology.
+  However, I must make a case for unlisting the functional characteristic. If I was an ontologist seeking to construct a knowledge base outside of a delimited domain     of one to one functional relationships and wanted to represent the class of all red things, then I ought to unlist the functional characteristic so that the object     property relation “characteristic of” could represent and infer a one to many relationship, which is not functional by definition, i.e. 0..1..X.
+  Thus, although the “characteristic of” object property has the functional characteristic listed likely for pragmatic purposes, I argue that it would be better to       dispense with it if one’s aim is to represent relations that are one to many, many to one, or many to many. 
+
   (e)  Inverse Functional
+  The object property, “has characteristic”, should not have the inverse functional property listed.
+  It’s definition is the inverse of “characteristic of” (from [d]). The “characteristic of” object property’s definition is: “a relation between a specifically           dependent continuant (the characteristic) and any other entity(the bearer), in which the characteristic depends on the bearer for its existence.”
+  This answer relies on what was said in (d), but is its inversion. 
+  Like the functional characteristic, the inverse functional property relation means that for any given individual, the property can have at most one value. In other     words, there can be at most one out going relationship along the property for that individual. This means that an inverse functional object property has a             multiplicity 0..1 attached to it. For example, it might be said that apples “has characteristic” color red. Or put in another way, this apple has this characteristic   color red.
```

Comment:
"the inverse functional property relation means that for any given individual, the property can have at most one value."

F(x)=y is a common function, where x is an argument of the function F, and y is a value of F. Contra what you said, that the relation P is inverse functional should mean: P can have at most one **argument**. So using your example of redness, if "has characteristic" is inverse functional, then for any redness, at most one bearer can bear it.